### ISSUE TITLE
Devportal API Tryout page accessibility issues

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#8379]
### Describe your problem(s)

1. Input Element has no accessible name, associated <label>, or [title].

![image](https://user-images.githubusercontent.com/20179540/84887276-22574280-b0b3-11ea-8b84-93401c40350c.png)

2. Get test key button is not focusable through the keyboard

3. tooltip  is not focusable through the keyboard

4. Schemes select has no accessible name, associated <label>, or [title].

![image](https://user-images.githubusercontent.com/20179540/84919530-a7585100-b0df-11ea-91dc-0146854ea5f2.png)

5. Text does not meet minimum AA contrast ratio (4.5:1).

![image](https://user-images.githubusercontent.com/20179540/84922901-22bc0180-b0e4-11ea-8c65-9ca63180a0e1.png)

6. collapse keyboard on focus state is not visible

![image](https://user-images.githubusercontent.com/20179540/84923633-12f0ed00-b0e5-11ea-93c5-e7c38fd8c0a7.png)

### How will you implement it

---
